main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsAttributionSchema.java

```diff
@@ -0,0 +1,39 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+@GtfsTable("attributions.txt")
+public interface GtfsAttributionSchema extends GtfsEntity {
+    @FieldType(FieldTypeEnum.ID)
+    @PrimaryKey
+    String attributionId();
+
+    @FieldType(FieldTypeEnum.ID)
+    @ForeignKey(table = "agency.txt", field = "agency_id")
+    String agencyId();
+
+    @FieldType(FieldTypeEnum.ID)
+    @ForeignKey(table = "routes.txt", field = "route_id")
+    String routeId();
+
+    @ForeignKey(table = "trips.txt", field = "trip_id")
+    String tripId();
+
+    @Required
+    String organizationName();
+
+    boolean isProducer();
```

Comment:
Should these "Enums" in GTFS data types be enums instead of boolean values? It's possible (although probably unlikely in this particular field case) that a `2` value could be added in the future. Looks like GTFS actually doesn't have a "Boolean" data type.

Reply:
Right, I've spent a lot of time choosing between boolean and enum. I agree that enums are better because they are flexible.